Tests/aweXpect.Reflection.Tests/ThatMethods.AreStatic.Tests.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using aweXpect.Reflection.Collections;
+using aweXpect.Reflection.Tests.TestHelpers.Types;
+
+namespace aweXpect.Reflection.Tests;
+
+public sealed partial class ThatMethods
+{
+	public sealed class AreStatic
```

Comment:
Please also add the static tests for the plural form (`AreStatic` / `AreNotStatic`) for constructors, fields and properties (as they are currently missing.